bazel/BUILD

```diff
@@ -211,6 +211,11 @@ config_setting(
     values = {"define": "log_debug_assert_in_release=enabled"},
 )
 
+config_setting(
+    name = "debug_known_issues",
```

Comment:
Can this be a bit more clear on naming? "ignore_known_issues_asserts" or something like that? And just make it a boolean? It's not clear what this does from a quick read. This also needs documenting in https://github.com/envoyproxy/envoy/tree/master/bazel#enabling-optional-features